src/components/Image/Image.tsx

```diff
@@ -6,8 +6,10 @@ import ImageElement from './ImageElement';
 type ImageProps = {
   alt: string;
   caption?: string;
```

Comment:
Now that we have `captionVariant`, maybe `caption` should become `captionText` to avoid confusion?